src/ClientTest.h

```diff
@@ -5,6 +5,9 @@
 
```

Comment:
Don't move this this file.

Reply:
Graph/Meta/Storage' client test all need this file,  so where should I put this file?

Reply:
Write another in storage test folder.

Reply:
ok